overlay/mobile-nixos/kernel/builder.nix

```diff
@@ -21,6 +21,7 @@
 , writeTextFile
 , writeShellScriptBin
 
+, lz4
```

Comment:
I would add it to the kernels building with `lz4`; `nativeBuildInputs` from kernels are appended.